README.md.template

```diff
@@ -234,6 +234,35 @@ At this time, Google hasn't shared any immediate plans to bring first-party supp
 - [Add support for JUnit 5 (issuetracker.google.com)](https://issuetracker.google.com/issues/127100532)
 - [JUnit 5 support (github.com/android/android-test)](https://github.com/android/android-test/issues/224)
 
+## Support for @Rules
+Since JUnit 5 has replaced the `@Rule` mechanism with Extensions, the following artifacts help bridge the gap until Android officially transitions to JUnit 5.
```

Comment:
```suggestion
## Support for @Rules

Since JUnit 5 has replaced the `@Rule` mechanism with Extensions, the following artifacts help bridge the gap until Android officially transitions to JUnit 5.
```

Super nit, blank line after headings please 🙇‍♂️